src/v/cluster/cluster_link/frontend.cc

```diff
@@ -156,6 +157,18 @@ ss::future<errc> frontend::update_cluster_link_configuration(
     co_return co_await do_mutation(std::move(c), timeout);
 }
 
+ss::future<errc> frontend::update_shadow_link_status(
```

Comment:
do we still need this API if we are using "force" boolean in DeleteShadowLink API?